odoo/addons/base/models/ir_ui_view.py

```diff
@@ -131,7 +132,10 @@ def simplify_modifiers(modifiers):
 def transfer_modifiers_to_node(modifiers, node):
     if modifiers:
         simplify_modifiers(modifiers)
-        node.set('modifiers', json.dumps(modifiers))
+        if modifiers:
+            node.set('modifiers', json.dumps(modifiers))
+        else:
+            node.attrib.pop('modifiers', None)
```

Comment:
Why? We shouldn't have a `modifiers` attribute on the nodes, so this should do nothing. Do we really want / need to guard against someone putting an explicit `modifiers` attribute on a node?

Reply:
I don't know if we should, I agree it shouldn't be the case, but technically it was the case before, because of all these `modifiers="{}"` that were put.

I can remove it if you believe this won't cause any issue.

Reply:
IDK I'm just wondering if we care. AFAIK all the `modifiers=` are in the JS tests but that's it.